src/server/ns_turn_maps.c

```diff
@@ -655,16 +668,31 @@ static addr_elem *addr_list_get(addr_list_header *slh, const ioa_addr *key) {
   return NULL;
 }
 
-static const addr_elem *addr_list_get_const(const addr_list_header *slh, const ioa_addr *key) {
+static addr_elem *addr_list_get_no_port(addr_list_header *slh, const ioa_addr *key) {
+  return _addr_list_get(slh, key, 0);
+}
+
+static addr_elem *addr_list_get(addr_list_header *slh, const ioa_addr *key) {
+  return _addr_list_get(slh, key, 1);
+}
```

Comment:
Adding an argument would require updating all references to `addr_list_get` in all of coturn.

I wanted to avoid making changes to the entire code-base to try to minimize changes. I am okay with updating all references to `addr_list_get` to include a port variable if you think this is better.

Let me know which way you want to go on this. Both are fine by me.